docs/index/index-management.md

```diff
@@ -18,51 +18,49 @@
 # CarbonData Index Management
 
 - [Overview](#overview)
-- [DataMap Management](#datamap-management)
+- [Index Management](#index-management)
 - [Automatic Refresh](#automatic-refresh)
 - [Manual Refresh](#manual-refresh)
-- [DataMap Catalog](#datamap-catalog)
-- [DataMap Related Commands](#datamap-related-commands)
+- [Index Catalog](#index-catalog)
+- [Index Related Commands](#index-related-commands)
   - [Explain](#explain)
-  - [Show DataMap](#show-datamap)
+  - [Show Index](#show-index)
 
 
 
 ## Overview
 
-DataMap can be created using following DDL
+Index can be created using following DDL
 
 ```
-CREATE DATAMAP [IF NOT EXISTS] datamap_name
-[ON TABLE main_table]
-USING "datamap_provider"
-[WITH DEFERRED REBUILD]
-DMPROPERTIES ('key'='value', ...)
-AS
-  SELECT statement
+CREATE INDEX [IF NOT EXISTS] index_name
+ON TABLE [db_name.]table_name (column_name, ...)
+AS carbondata/bloomfilter/lucene
+[WITH DEFERRED REFRESH]
+[PROPERTIES ('key'='value')]
 ```
 
-Currently, there are 5 DataMap implementations in CarbonData.
+Currently, there are 3 Index implementations in CarbonData.
 
-| DataMap Provider | Description                              | DMPROPERTIES                             | Management       |
+| Index Provider | Description                              | PROPERTIES                             | Management       |
 | ---------------- | ---------------------------------------- | ---------------------------------------- | ---------------- |
-| mv               | multi-table pre-aggregate table          | No DMPROPERTY is required                | Manual/Automatic           |
+| secondary-index  | secondary-index tables to hold blocklets as indexes and managed as child tables        | No PROPERTY is required                | Automatic           |
```

Comment:
Can remove Properties column,  as now, index_columns will be provided in CREATE statement and not required to give in properties